engine.py

```diff
@@ -88,7 +88,7 @@ def get_bishop_moves(self, r, c, moves):
 		"""
 
 		##TODO
-		direction = ((-1,-1), (1,1), (1,-1), (-1,1)) # possible direction
+		direction = ((-1,-1), (1,1), (1,-1), (-1,1)) # possible  Bishop direction
 		if self.light_to_move:
```

Comment:
Add a comment to this line, and line 107, and put a few blank lines (I suggest two) between line 106 and 107